examples/vkload.cpp

```diff
@@ -116,9 +102,7 @@ Texture::cleanup()
     */
 }
 
-void
-Texture::prepareSamplerAndView()
-{
+void Texture::prepareSamplerAndView() {
```

Comment:
So this is an on/off option. I used return type on separate line for the libktx code because that was the style, but used return type on same line for newer things like the ktx CLI tools which use more modern style. If you could provide a list of directories where you'd like us to retain the legacy "return type on separate line" style, we can just configure things accordingly.